### PR TITLE
Added cheat for metaclasses

### PR DESCRIPTION
**Pull request by theshortman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4595,6 +4596,61 @@ It is mandatory to define the data type in dataclass. However, If you don't want
...
+```python
+>>> from dataclasses import dataclass
+>>> from typing import Any
+
+>>> @dataclass
+... class WithoutExplicitTypes:
+...    name: Any
+...    value: Any = 42
+...
+```
```
Comment (StephenBrown2) [inline README.md:4608]: This looks to be duplicated from the section above?

File: README.md [hunk 2]
```diff
@@ -4595,6 +4596,61 @@ It is mandatory to define the data type in dataclass. However, If you don't want
...
+
+## Metaclasses
+
+`Metaclasses` let you change the class automatically.
```
Comment (StephenBrown2) [inline README.md:4612]: I don't think this is correct... I'm not sure what is meant by the sentence.

File: README.md [hunk 3]
```diff
@@ -4595,6 +4596,61 @@ It is mandatory to define the data type in dataclass. However, If you don't want
...
+... 
+>>> class MessageClass(metaclass=MessageMetaclass):
+...     def print_message(self):
+...         print(self.message)
+... 
+>>> MessageClass().print_message()
+hello
+```
+
+The main use case for a metaclass is creating an API (Django).
```
Comment (StephenBrown2) [inline README.md:4650]: This is not correct. Metaclasses have many uses, and there is no example of "creating an API" here, so it feels irrelevant.

File: README.md (hunk 1 above)
Comment (theshortman) [reply README.md:4608]: I agree. This is my failure.

**Comment by wilfredinni** [comment]
@theshortman I really appreciate your pull request, but as @StephenBrown2 mentioned there are some sentences that don't make much sense if they aren't followed by examples or further explanations. Remember that this cheat sheet is supposed to be beginner-friendly.
